test/controllers/project_category_controller_test.exs

```diff
@@ -52,106 +30,62 @@ defmodule CodeCorps.ProjectCategoryControllerTest do
       project_category = insert(:project_category, project: project, category: category)
 
       conn
-      |> get(project_category_path(conn, :show, project_category))
+      |> request_show(project_category)
       |> json_response(200)
       |> Map.get("data")
       |> assert_result_id(project_category.id)
-      |> assert_jsonapi_relationship("project", project.id)
-      |> assert_jsonapi_relationship("category", category.id)
     end
 
-    test "does not show resource and instead throw error when id is nonexistent", %{conn: conn} do
-      path = conn |> project_category_path(:show, -1)
-      assert conn |> get(path) |> json_response(:not_found)
+    test "renders 404 when id is nonexistent", %{conn: conn} do
+      assert conn |> request_show(:not_found) |> json_response(404)
     end
   end
 
   describe "create" do
-    @tag :authenticated
-    test "creates and renders resource when data is valid", %{conn: conn, current_user: current_user} do
+    @tag authenticated: :admin
+    test "creates and renders resource when data is valid", %{conn: conn} do
       organization = insert(:organization)
       category = insert(:category)
       project = insert(:project, organization: organization)
-      insert(:organization_membership, role: "admin", member: current_user, organization: organization)
-
-
-      payload = build_payload |> put_relationships(project, category)
 
-      path = conn |> project_category_path(:create)
-      response = conn |> post(path, payload) |> json_response(201)
-      data = response |> Map.get("data")
-
-      data
-      |> assert_jsonapi_relationship("project", project.id)
-      |> assert_jsonapi_relationship("category", category.id)
-
-      project_category = ProjectCategory |> Repo.get(data["id"])
-      assert project_category
-      assert project_category.project_id == project.id
-      assert project_category.category_id == category.id
+      attrs = %{category: category, project: project}
+      assert conn |> request_create(attrs) |> json_response(201)
     end
 
     @tag authenticated: :admin
-    test "does not create resource and renders 422 when data is invalid", %{conn: conn} do
-      payload = build_payload()
-
-      path = conn |> project_category_path(:create)
-      data = conn |> post(path, payload) |> json_response(422)
-      assert data["errors"] != %{}
+    test "renders 422 when data is invalid", %{conn: conn} do
+      invalid_attrs = %{}
```

Comment:
Gotcha. Do we need to even pass anything in the tests below, then? Can we just do `conn |> request_create |> json_response(401)` ?